components/comment/modal/modal.vue

```diff
@@ -39,10 +39,6 @@
 
 >>> .timeline-list-item-split
 	full-bleed()
-
-// The full-bleed would add a scrollbar if we didn't cut it off like this.
-.modal-body
```

Comment:
This was apparently to prevent a horizontal scrollbar to appear on mobile with children of the modal bleeding out of it, with the modal already being 100% width.

That seems to not happen anymore, so removing this does not affect it.